support/client/lib/vwf/model/jPlayer.js

```diff
@@ -403,7 +432,7 @@ define( [
                     }
                     break;
                 case "video":
-                    if ( url.search( "data:video/mp4" ) === 0 ) {
+                    if ( url.search( "data:video/mp4" ) === 0 || url.search( ".mp4$" ) > -1 ) {
```

Comment:
It expects "data:video/mp4" to be a substring of index 0 of the URL, whereas I'm matching a ".mp4" at the end of the string, which is the purpose of the '$'. > -1 means that a match exists.